scripts/master_voting/vote.js

```diff
@@ -0,0 +1,105 @@
+// Signs a master proposal using sr25519, prints the signature as hex.
+
+import { u8aToHex } from '@polkadot/util';
+import { connect, keypair } from '../helpers';
+
+const fsp = require('fs').promises;
+
+const USAGE =
+`
+Use:
+  ./vote.js <round_no> <./path/to/proposal.json>[ yes]
+
+Call this script first without "yes" to perform a dry-run. The name of the call will be printed as \
+well as its arguments. The provided <round_no> will be checked against the running node.
+
+If you wish to vote yes on the proposal re-run this script with "yes", and a signature will be \
+generated.
+
+Expected Env vars:
+- FullNodeEndpoint
+    Websocket rpc url to a trusted live node e.g. wss://example.com:9944
+- MasterMemberSecret
+    Secret phrase or hex encoded private key with which to sign. Not required for dry-run.
+`;
+
+main().catch(e => {
+  console.error(e);
+  process.exit(1);
+}).then(_ => {
+  process.exit(0);
+});
+
+async function main() {
```

Comment:
You'll notice the program structure of this script is slightly different from that of `scripts/master_voting/submit.js`. There isn't a great technical reason for the difference. If it's a problem let me know and I'll rewrite `scripts/master_voting/vote.js` in the style of `scripts/master_voting/submit.js`.